service/src/main/java/com/ge/predix/acs/config/CloudDataSourceConfig.java

```diff
@@ -39,7 +38,6 @@
  */
 @Configuration
 @Profile({ "cloudDbConfig" })
-@EnableAutoConfiguration
```

Comment:
We do rely on auto configuration, although using @SpringBootApplication once is enough. No need to repeat @EnableAutoConfiguration all throughout the code. 

http://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-using-springbootapplication-annotation.html